File: Python/GuessTheNumber.py
from random import randint

def main():  
    n = randint(1, 100)

    w = False
    i = 0
    while w == False:    
        i += 1
        print("Podaj liczbe: ", end='')
        l = int(input())
        
        if l > n:
            print("Za duzo!")
        elif l < n:
            print("Za malo!")        
        else:        
            print("Wygrales!")
            print("Za ",i, " razem" )
            w = True

if __name__ == "__main__":
    main()

File: C/GuessTheNumber.c
#include <stdlib.h>
#include <stdio.h>
#include <time.h>

int main()
{
    srand(time(NULL));

    int attempts = 1;
    bool win = false;
    int number = rand() % 100 + 1;

    do
    {
        int guess;
        printf("Podaj liczbe: ");
        scanf("%d", &guess);
        
        if (guess > number)
        {
            printf("%d proba: za duzo!\n", &attempts);
            attempts++;
        }
        else if (guess < number)
        {
            printf("%d proba: za malo!\n");
            attempts++;
        }
        else
        {
            printf("Gratulacje: Wygrales w %d probie!\n");
            win = true;
        }

    }
    while (win == false);
}